src/System.Net.Security/ref/TlsCipherSuite.tt

```diff
@@ -0,0 +1,133 @@
+﻿<#@ template debug="false" hostspecific="false" language="C#" #>
+<#@ assembly name="System.Core" #>
+<#@ assembly name="System.Xml" #>
+<#@ assembly name="System.Xml.Linq" #>
+<#@ import namespace="System.Linq" #>
+<#@ import namespace="System.Text" #>
+<#@ import namespace="System.Collections.Generic" #>
+<#@ import namespace="System.Xml.Linq" #>
+<#@ output extension=".cs" #>
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// This file has been auto-generated. Do not edit by hand.
+// Instead open Developer Command prompt and run: TextTransform FileName.tt
```

Comment:
Does this work on non-Windows?

Reply:
Unfortunatelly no :-( We already use .tt files in other places though and since they require changing APIs we will probably want someone internal do it anyway...